src/routes/(site)/teams/officer-profile.svelte

```diff
@@ -31,6 +32,7 @@
   <div>
     <h3 class="acm-heavier">{officerName}</h3>
     <p class="acm-heavy">{@html titleHTML}</p>
+    <p>@{officerDiscord}</p>
```

Comment:
Copy the Discord username when clicked. Change the style on hover. 

- Prior art: <https://github.com/EthanThatOneKid/acmcsuf.com/blob/134d89cef204ae3d61a5f2494152139ebc63443c/src/routes/(site)/colors/color.svelte#L17>

Reply:
Sweet! Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/1099/commits/55b173bc7dc5084613ef5715faa825fdeac47efd>.